src/android/java/io/jxcore/node/BtConnectorHelper.java

```diff
@@ -5,17 +5,20 @@
 import android.content.Context;
```

Comment:
@DrJukka Please do me a favor. Follow the team read me instructions to set up ThaliTest. Then open up IntelliJ and go to file->open and point it to the build.gradle file in ThaliTest/platforms/android. Then go to Analyze->Inspect Code and choose custom scope, choose local and then select the android/src directory and hit 'Include Recursively'. You should see a pattern of the form `file[android]:src//*` then hit OK and run the analysis. Then start looking through the analysis. Not everything it calls out is relevant (for example, it catches some issues with JXcore) and not all of its suggestions are necessary but it found a bunch of really obvious errors. I would appreciate it if you could run it so that I don't have to even think about those classes of errors and can just look at the code. Thanks!

Oh, one more thing. Before you run the analysis go to file->project structure->project and set the Project SDK to Android and set the project language level to 7. This will prevent some spurious suggestions.

But please do this. It will clean up spelling errors. Inefficient Ifs. Spurious variables. Unused methods. Etc. Having all this cleaned up before a code review makes for a much more efficient code review.

But remember, suggestions aren't orders and some of their suggestions can make things worse. You have to keep your wits about you and make sure that their suggestions actually make sense given your goals for the code. This is not a mechanical exercise. Think of it rather as hints. You need to think if you agree with the hints that IntelliJ gives.

Thanks!